src/core/lib/security/authorization/rbac_translator.cc

```diff
@@ -394,7 +392,8 @@ ParseAuditLogger(const Json& json, size_t pos) {
         absl::StrFormat("\"audit_loggers[%d].name\" is not a string.", pos));
   }
   absl::string_view name = it->second.string();
-  Json config = Json::Object();
+  // The config defaults to an empty object.
+  Json config = Json::FromObject(Json::Object());
```

Comment:
Can just say `Json::FromObject({})`.  The compiler is smart enough to know the type of the `{}`.

Reply:
Changed.